### ISSUE TITLE
Actions when event starts

### ISSUE THREAD

**Issue by imeed166** [davish/obsidian-full-calendar#50]
Sorry if this is not in the scope of this plugin.
The idea is to do something when the event start, such actions include :
- notification pop up
- executing templater scripts 
- other actions that I haven't thought of

**Comment by davish** [comment]
I don't think this is in scope for now. But I'll leave this open for people to register interest and give suggestions!
